package.json

```diff
@@ -115,7 +115,7 @@
   },
   "homepage": "https://www.fastify.io/",
   "devDependencies": {
-    "@fastify/ajv-compiler-8": "github:fastify/ajv-compiler#ajv-8",
+    "@fastify/ajv-compiler-8": "npm:@fastify/ajv-compiler@^2.0.0",
```

Comment:
do we need this?

Reply:
This alias let us running tests with the ajv8 module, so it lets us spot if the integration is still working.

Before it was pointing to a defined branch in the `fastify/ajv-compiler` project. With the latest release, we can rely on the npm registry

The test in place into `main` is this one:

https://github.com/fastify/fastify/blob/e3903cfaf571dd6c24be3dc2965233869d90e5be/test/schema-special-usage.test.js#L13